model-uploader/src/main/java/ai/djl/uploader/ExporterArguments.java

```diff
@@ -94,16 +107,16 @@ public static Options getOptions() {
                         .addOption(new Option("model", false, "import general model"));
         og.setRequired(true);
         options.addOptionGroup(og);
-        options.addRequiredOption(
-                "an",
-                "artifact-name",
-                true,
-                "the name of the artifact or the name of the model to import");
         options.addRequiredOption(
                 "ai",
                 "artifact-id",
                 true,
                 "the way to categorize the artifact, for example, resnet50 has resnet as artifact id");
+        options.addOption(
```

Comment:
If user import Keras model without specify the name. Saying he has a file stored in:

`path/my_model.h5`

Then the output of the metadata will be `null.h5`.

Reply:
if artifact-name is not specified, it will be set to artifact-id.
user has to specify artifact-name if it's different from artifact-id